docs/content/preview/integrations/atomicjar.md

```diff
@@ -0,0 +1,111 @@
+---
+title: AtomicJar Testcontainers
+linkTitle: AtomicJar Testcontainers
+description: Use AtomicJar Testcontainers with YSQL API
+menu:
+  preview_integrations:
+    identifier: ysql-atomicjar
+    parent: integrations
+    weight: 571
+type: docs
+---
+
+[AtomicJar Testcontainers](https://www.testcontainers.org/) is a Java library that supports JUnit tests, providing lightweight, throwaway instances of common services that can run in a Docker container. Though initially targeted at the JVM ecosystem, it can be leveraged for other programming languages. Testcontainers is a great way to write integration tests for frameworks like Spring Boot, with a real database.
+
+YugabyteDB supports for the [YugabyteDB module](https://www.testcontainers.org/modules/databases/yugabytedb/) in Testcontainers.
+
+This document describes how to use Testcontainers to write integration tests for a Spring Boot application with YugabyteDB .
+
+## Prerequisites
+
+To use Testcontainers, ensure that you have the following:
+
+- Java Development Kit (JDK) 17 or later installed. You can use [SDKMAN](https://sdkman.io/install) to install the JDK runtime.
+
+- Docker Engine. To download and install Docker, select one of the following environments:
+
+    <i class="fa-brands fa-apple" aria-hidden="true"></i> [Docker for Mac](https://store.docker.com/editions/community/docker-ce-desktop-mac)
+
+    <i class="fa-brands fa-centos"></i> [Docker for CentOS](https://store.docker.com/editions/community/docker-ce-server-centos)
+
+    <i class="fa-brands fa-ubuntu"></i> [Docker for Ubuntu](https://store.docker.com/editions/community/docker-ce-server-ubuntu)
+
+    <i class="icon-debian"></i> [Docker for Debian](https://store.docker.com/editions/community/docker-ce-server-debian)
+
+## Build the Testcontainer project
+
+Refer to the [Spring Boot source](https://github.com/yugabyte/yb-ms-data/tree/main/springboot) from GitHub for your local setup. The source consists of a basic JPA-based web application with CRUD functionality.
+
+You need to build the Testcontainer project using the following steps:
+
+1. Define the library dependency in the build file. Refer to the `build.grade` file for complete details.
+
+    ```conf
```

Comment:
Is conf right? there's no coloring

Reply:
changed it to properties. That shows coloring. Pulled conf from some existing files. We'll have to change them sometime.